cogs/numb.py

```diff
@@ -0,0 +1,87 @@
+'''This is a guess the number game'''
+import random
+import asyncio
+from discord.ext import commands
+
+
+
+class NumbGuess:
+	'''this is a number guessing game'''
+	def __init__(self, bot):
+		self.bot = bot
+	@commands.command(name="numguess", pass_context=True)
+	async def mycom(self, context):
+	    '''this will start a number guessing game'''
+	    GuessesTaken = 0
+	    await self.bot.say("how hard would you like the number game to be? (easy/mid/hard)")
+	    mode = self.bot.wait_for_message(channel=context.message.channel, timeout=30)
+            if mode == 'easy': # pylint: disable=syntax-error
+                number = random.randint(1,10)
+
+	    elif mode == 'mid':
+		number = random.randint(1,50)
+
+	    elif mode == 'hard':
+		number = random.randint(1,100)
+
+	    #easter egg
+	    elif mode == 'no u':
+		number = random.randint(1,420)
+		await self.bot.say('no u')
+	    else:
+		await self.bot.say('type again')
+                while True:
```

Comment:
Wouldn't this cause the bot to continuously ask to type again even if it timeouts?